root/edit/details/EditWork.js

```diff
@@ -0,0 +1,159 @@
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import WordDiff from '../../static/scripts/edit/components/edit/WordDiff';
+import {INSERT, DELETE} from '../../static/scripts/edit/utility/editDiff';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import DiffSide from '../../static/scripts/edit/components/edit/DiffSide';
+import FullChangeDiff from
+  '../../static/scripts/edit/components/edit/FullChangeDiff';
+import {commaOnlyListText} from
+  '../../static/scripts/common/i18n/commaOnlyList';
+
+type Props = {
+  ...EditT,
+  +data: {
+    new: {
+      attributes?: {
+        [attributeName: string]: $ReadOnlyArray<WorkAttributeT>,
+      },
+      languages: $ReadOnlyArray<string>,
+    },
+  },
+  +display_data: {
+    attributes?: {
+      [attributeName: string]: CompT<$ReadOnlyArray<string>>,
+    },
+    comment?: CompT<string | null>,
+    iswc?: CompT<string | null>,
+    language?: CompT<LanguageT | null>,
+    languages?: CompT<$ReadOnlyArray<string>>,
+    name?: CompT<string>,
+    type?: CompT<WorkTypeT | null>,
+    work: WorkT,
+  },
+};
+
+const EditWork = ({edit}: {edit: Props}) => {
+  const display = edit.display_data;
+  const comment = display.comment;
+  const iswc = display.iswc;
+  const name = display.name;
+  const language = display.language;
+  const languages = display.languages;
+  const attributes = display.attributes ?? {};
+  console.log(edit.data.new);
+  return (
+    <table className="details edit-work">
+      <tr>
+        <th>{addColonText(l('Work'))}</th>
+        <td colSpan="2">
+          <DescriptiveLink entity={display.work} />
+        </td>
+      </tr>
+      {name ? (
+        <WordDiff
+          label={addColonText(l('Name'))}
+          newText={name.new}
+          oldText={name.old}
+        />
+      ) : null}
+      {comment ? (
+        <WordDiff
+          label={addColonText(l('Disambiguation'))}
+          newText={comment?.new ?? ''}
+          oldText={comment?.old ?? ''}
+        />
+      ) : null}
+      {iswc ? (
+        <Diff
+          label={addColonText(l('ISWC'))}
+          newText={iswc?.new ?? ''}
+          oldText={iswc?.old ?? ''}
+        />
+      ) : null}
+      {display.type ? (
+        <FullChangeDiff
+          label={addColonText(l('Work type'))}
+          newContent={lp_attributes(display
+            .type?.new?.name ?? '', 'work_type')}
+          oldContent={lp_attributes(display
+            .type?.old?.name ?? '', 'work_type')}
+        />
+      ) : null}
+      {language ? (
+        <FullChangeDiff
+          label={addColonText(l('Language'))}
+          newContent={language.new?.name ?? ''}
+          oldContent={language.old?.name ?? ''}
```

Comment:
Still wondering why it wasn't done in the AddWork.js file though if language was also to be translated 🤔

Reply:
Because I missed that you didn't do it (in fact, I left a comment but you seemingly didn't make the change) :) A lot of our old TT edit templates were untranslated, which is bad, but we might as well fix them while we're porting them.

Reply:
Cool. I'll get it done in sometime